src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2Engine.java

```diff
@@ -137,131 +59,93 @@
         samplesList = new IndexedSampleList(new ArrayList<>(ReadUtils.getSamplesFromHeader(header)));
-        if (!samplesList.asListOfSamples().contains(tumorSampleName)) {
+        if (!samplesList.asListOfSamples().contains(MTAC.tumorSampleName)) {
             throw new UserException.BadInput("BAM header sample names " + samplesList.asListOfSamples() + "does not contain given tumor" +
-            " sample name " + tumorSampleName);
-        } else if (normalSampleName != null && !samplesList.asListOfSamples().contains(normalSampleName)) {
+                    " sample name " + MTAC.tumorSampleName);
+        } else if (MTAC.normalSampleName != null && !samplesList.asListOfSamples().contains(MTAC.normalSampleName)) {
             throw new UserException.BadInput("BAM header sample names " + samplesList.asListOfSamples() + "does not contain given normal" +
-                    " sample name " + normalSampleName);
+                    " sample name " + MTAC.normalSampleName);
         }
 
         //If the samples specified are exactly one normal and one tumor, use the TCGA VCF sample header format
-        printTCGAsampleHeader = samplesList.numberOfSamples() == 2 && normalSampleName != null;
+        printTCGAsampleHeader = samplesList.numberOfSamples() == 2 && MTAC.normalSampleName != null;
 
-        final VariantAnnotatorEngine annotationEngine = initializeVCFOutput();
-        referenceReader = AssemblyBasedCallerUtils.createReferenceReader(referenceArguments.getReferenceFileName());
-        assemblyEngine = MTAC.assemblerArgs.createReadThreadingAssembler(MTAC.DEBUG, MTAC.MIN_BASE_QUALTY_SCORE);
-
-        MIN_TAIL_QUALITY = (byte)(MTAC.MIN_BASE_QUALTY_SCORE - 1);
+        if (MTAC.ENABLE_CLUSTERED_READ_POSITION_FILTER) {
+            MTAC.annotationsToUse.add("ClusteredReadPosition");
+        }
+        annotationEngine = VariantAnnotatorEngine.ofSelectedMinusExcluded(MTAC.annotationGroupsToUse,
+                MTAC.annotationsToUse,
+                MTAC.annotationsToExclude,
+                MTAC.dbsnp.dbsnp,
+                MTAC.comps);
 
-        // setup the likelihood calculation engine
-        if ( MTAC.likelihoodArgs.phredScaledGlobalReadMismappingRate < 0 ) MTAC.likelihoodArgs.phredScaledGlobalReadMismappingRate = -1;
+        assemblyEngine = AssemblyBasedCallerUtils.createReadThreadingAssembler(MTAC);
 
-        log10GlobalReadMismappingRate = AssemblyBasedCallerUtils.calculateLog10GlobalReadMismappingRate(MTAC.likelihoodArgs.phredScaledGlobalReadMismappingRate);
+        MIN_TAIL_QUALITY = (byte)(MTAC.minBaseQualityScore - 1);
 
-        likelihoodCalculationEngine = AssemblyBasedCallerUtils.createLikelihoodCalculationEngine(MTAC.likelihoodArgs, log10GlobalReadMismappingRate);
+        if ( MTAC.likelihoodArgs.phredScaledGlobalReadMismappingRate < 0 ) MTAC.likelihoodArgs.phredScaledGlobalReadMismappingRate = -1;
```

Comment:
Are you sure?  I can't find any mention of "phred" in Mutect2Engine.java in the latest commit.